src/components/filterBar/view/filterBarStyles.js

```diff
@@ -1,4 +1,5 @@
 import EStyleSheet from 'react-native-extended-stylesheet';
+import { relative } from 'path';
```

Comment:
I guess you imported them by mistake?

Reply:
yes likely it was already imported, will remove